docs/src/install/index.md

```diff
@@ -223,43 +223,120 @@ The packages can be viewed directly from the [Cloudsmith.io](https://cloudsmith.
 </table>
 
 ## Shell completions
+
 :::note
 Shell-completions are supported with `tedge` >= 1.5.0
 :::
 
-To make using the CLI easier, `tedge` provides tab-completion support. This can
-be enabled by adding a small configuration to your shell environment:
+To make using the Command Line Interface (CLI) easier, `tedge` provides shell completion support (also known as tab-completions). Shell completions allow users to discover commands and configuration directly on the command line without having to lookup up online documentation.
```

Comment:
One "up" should be removed I suppose.

```suggestion
To make using the Command Line Interface (CLI) easier, `tedge` provides shell completion support (also known as tab-completions). Shell completions allow users to discover commands and configuration directly on the command line without having to look up online documentation.
```

Reply:
I also wasn't sure whether it should be "lookup" or "look up"...so I changes it to "check"